### PR TITLE
Remove Shapeshift (.ai & .io)

### PR DESCRIPTION
**Pull request by Cactii1**
Shapeshift.ai no longer operating and Shapeshift.io de-listed Monero

### CODE REVIEW

File: _data/merchants.yml [hunk 1]
```diff
@@ -83,10 +83,6 @@
       url: https://poloniex.com/exchange/btc_xmr
     - name: REXI.cc
       url: https://rexi.cc
-    - name: ShapeShift (Instant)
-      url: https://shapeshift.io/
-    - name: SideShift AI - A rapid coin swap that fully supports Monero
-      url: https://sideshift.ai
```
Comment (erciccione) [inline _data/merchants.yml:89]: Sideshift seems to be unrelated to shapeshift, and does list Monero.

**Review by erciccione — CHANGES_REQUESTED** [review]

**Review by erciccione — COMMENTED** [review]
Looks ok but the commit message is quite unclear. We are removing only shapeshift.io (not .ai). Something like `remove shapeshift.io and sideshift.ai`

**Comment by Cactii1** [comment]
Ah jeese... sorry.. My bad, totally read that wrong. I need to be more careful.
